src/frontend/next/src/components/ScrollAction/ScrollAction.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC, ReactChildren, MouseEvent } from 'react';
+
+type ScrollActionProps = {
+  children: ReactChildren;
+};
+
+const ScrollAction: FC<ScrollActionProps> = ({ children }) => {
+  const handleClick = (event: MouseEvent<HTMLElement>) => {
+    const mobile = window.innerWidth <= 1065;
+    const anchor = mobile
+      ? ((event.target as HTMLElement).ownerDocument || document).querySelector(
```

Comment:
Can you not use `event.currentTarget` here so you don't have to cast?

Reply:
Yeah that was what I tried initially, but since I also tried to keep everything the same so I cast it. Should I use currentTarget instead?